moveit_experimental/moveit_jog_arm/src/low_pass_filter.cpp

```diff
@@ -98,4 +101,17 @@ double LowPassFilter::filter(double new_measurement)
 
   return new_filtered_measurement;
 }
+
+void LowPassFilter::updateFilterCoeff(double new_filter_coeff)
+{
+  if (new_filter_coeff >= 1)
+  {
+    scale_term_ = 1. / (1. + new_filter_coeff);
+    feedback_term_ = 1. - new_filter_coeff;
+  }
+  else
+  {
+    ROS_WARN_STREAM_NAMED(LOGNAME, "Filter coefficient < 1. makes the lowpass filter unstable. Ignoring request.");
```

Comment:
This behavior is different from what is in the constructor.  I suggest just calling this from the constructor so there is only one behavior and move the logic from the constructor here.